### PR TITLE
(REVIEW ONLY) some small enhancements

### PR DESCRIPTION
**Pull request by mmoll**
@stbenjam please comment, I might add more commits after actually having played around with it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# RVM
+.rvmrc
+.ruby-version
+.ruby-gemset
+
+# Editors
+tags
+.idea
+*.sw?
```
Comment (stbenjam) [inline .gitignore:19]: +1,000,000,000 - I kept committing .swp's the other day when I was working on the API. Can you send the same to `foreman_salt`?!

File: Rakefile [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+    require 'hammer_cli_foreman_salt/version'
+    require 'hammer_cli_foreman_salt/i18n'
     require 'gettext/tools'
 
-    domain = HammerCLIForemanBootdisk::I18n::LocaleDomain.new
-    GetText.update_pofiles(domain.domain_name, domain.translated_files, "#{domain.domain_name} #{HammerCLIForemanBootdisk.version.to_s}", :po_root => domain.locale_dir)
+    domain = HammerCLIForemanSalt::I18n::LocaleDomain.new
+    GetText.update_pofiles(domain.domain_name, domain.translated_files, "#{domain.domain_name} #{HammerCLIForemanSalt.version.to_s}", :po_root => domain.locale_dir)
   end
 end
```
Comment (stbenjam) [inline Rakefile:14]: We should also run this rake task if we can to create the needed po files

Maybe someday we want to translate it to German :-)

File: config/foreman_salt.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+:foreman_salt:
+    :enable_module: true
```
Comment (stbenjam) [inline config/foreman_salt.yml]: 2 spaces indentation is better I think

File: .gitignore (hunk 1 above)
Comment (mmoll) [inline .gitignore:19]: just add it in your api branch?

File: Rakefile (hunk 2 above)
Comment (mmoll) [inline Rakefile:14]: I use software only in English, so no idea how it's really supposed to work...

File: config/foreman_salt.yml (hunk 3 above)
Comment (mmoll) [inline config/foreman_salt.yml]: yup

**Comment by mmoll** [comment]
@stbenjam if it's OK for you I would squash and merge this and then release 0.0.2 of the gem. I already prepared a Debian package.

**Comment by stbenjam** [comment]
@mmoll Sounds good to me! :+1:

**Comment by stbenjam** [comment]
Wrong button :-)

**Comment by mmoll** [comment]
thanks, merged as 7b3c085 and tagged as 0.0.2. :1234:

**Pull request closed without merging**